### DISCUSSION TITLE
New descriptions for manuscripts of Sane Golgota, Nagare Maryam

### DISCUSSION THREAD

**Discussion by Getinet309** [SChAth/ScAthiop#1833] [Q&A] [answered]
I would like to create an XML file for  Sane Golgota, Nagare Maryam, and others which are found in Ethiopia at Dabra Golgota Dəbəq Gasha monastery which is near Qaranyo MadhaneAlem. 
These manuscripts don't have any digital records before and I can digitalize them and I can make a catalog for them also.

**Comment by thea-m** [comment, marked as answer]
Dear Getinet, 
Thank you for your issue. I apologise for the delay.
We make a difference between manuscripts and works. Manuscripts are the physical objects and works are the texts contained in them. For both, we create XML records: manuscript records and work records. 
You have already created successfully the manuscript record for DGDG001.
This manuscript record will contain references to the works contained in the manuscript. You can search for work IDs using the general search on BM or in the work view https://betamasaheft.eu/works/list. For example, if you type "Sane golgota" in https://betamasaheft.eu/works/list, the result will be a link to https://betamasaheft.eu/works/LIT2246salotz/main, the page of the record for this work. Its ID is LIT2246salotz, and you will refer to this ID in the manuscript record DGDG001 as `<title type="complete" ref="LIT2246salotz"/>`. 
Does that help you? 
Let's practice by trying to find the work IDs for the other works contained in this manuscript. It may help your searching to have a look at the transliteration principles we use: https://betamasaheft.eu/Guidelines/?id=transliteration-principles.
Please tell us how it goes!

**Reply by Getinet309** [reply]
Dear Dorothea,
Thank you very much for your helpful information. 
The manuscript record DGDG001 is not for https://betamasaheft.eu/works/LIT2246salotz/main, since  the physical description and the repository are different but the work is the same. can I use https://betamasaheft.eu/works/LIT2246salotz/main, as reference.
Thank You.

**Reply by PietroLiuzzo** [reply]
I have the feeling here we are mixing up Manuscripts and Textual units. DGDG001 is a manuscript, a physical object, with a repository and physical description. LIT2246salotz identifies a Work, a textual unit, which is contained in DGDG001. Please @Getinet309 make sure you are acquainted with this core distinction. You can start from here. https://betamasaheft.eu/Guidelines/?id=general

**Comment by PietroLiuzzo** [comment]
additionally, you could use the search functionality and look for ["Sane golgota" ](https://betamasaheft.eu/facet.html?query=%22Sane+golgota%22) and eventually filter to see only textual units
